subsys/spm/secure_services.c

```diff
@@ -47,6 +53,34 @@ int spm_secure_services_init(void)
 }
 
 
+#ifdef CONFIG_SPM_SERVICE_READ
+__TZ_NONSECURE_ENTRY_FUNC
+int spm_request_read(void *destination, const void *source, size_t num)
+{
+	static const u32_t read_ranges[] = {
```

Comment:
Just wondering, what happens when `PM_MCUBOOT_ADDRESS` is not defined and this array has size zero? Does the compiler get angry?

Reply:
No, when the SPM sample is built stand-alone, it does not have this define, and that works fine (this is part of CI).